app/main.go

```diff
@@ -2,14 +2,62 @@ package main
 
 import (
 	"fmt"
-	"github.com/ministryofjustice/opg-modernising-lpa/internal/fake"
+	"github.com/ministryofjustice/opg-go-common/env"
+	"html/template"
+	"log"
+	"net/http"
```

Comment:
if you set up https://pkg.go.dev/golang.org/x/tools/cmd/goimports instead of `gofmt` it will nicely sort and break up the imports

Reply:
I had this in the pre-commit hook setup but then realised on my last commit it hadn't been running. Looks like its working now though (had to re-install it) so I'll fix this up 👍